catalogue/webapp/services/prismic/fetch/index.ts

```diff
@@ -0,0 +1,90 @@
+import * as prismic from '@prismicio/client';
+import fetch from 'node-fetch';
+import { gql, GraphQLClient } from 'graphql-request';
+import { PrismicApiError } from '../types';
+import { capitalize } from '@weco/common/utils/grammar';
+
+export type GetServerSidePropsPrismicClient = {
```

Comment:
I don't think this is used anywhere within this project, but should it be?

Reply:
No, I was going to use something from content/services/prismic but ended up changing my mind